doc/chsmack.8

```diff
@@ -139,6 +139,13 @@ while still ensuring that the config files of the editor remain protected.
 
 This label is stored in the \fBsecurity.SMACK64TRANSMUTE\fR extended attribute.
 
+.TP
+.B -D, --drop
+
+Use this option to remove the smack properties that are not set for files.
```

Comment:
Now I see that you have added an example below. In that case, please skip the part of my comment about example.